browser-test/src/support/applicant_questions.ts

```diff
@@ -202,6 +203,52 @@ export class ApplicantQuestions {
     expect(tableInnerText).not.toContain(programName)
   }
 
+  async gotoApplicantHomePage() {
+    await this.page.goto(BASE_URL)
+    await waitForPageJsLoad(this.page)
+  }
+
+  async expectPrograms({
+    notStartedPrograms,
+    inProgressPrograms,
+    submittedPrograms,
+  }: {
+    notStartedPrograms: string[]
+    inProgressPrograms: string[]
+    submittedPrograms: string[]
+  }) {
+    const notStartedProgramNames = await this.programNamesForSection(
+      'Not started',
+    )
+    const inProgressProgramNames = await this.programNamesForSection(
+      'In progress',
+    )
+    const submittedProgramNames = await this.programNamesForSection('Submitted')
+
+    // Sort results before comparing since we don't care about order.
+    notStartedProgramNames.sort()
```

Comment:
The naming is a little off here since both are "Names" but only one includes it.

Use want and got instead?

Reply:
Done.